warehouse/migrations/versions/0cb51a600b59_rename_provider_to_publisher.py

```diff
@@ -0,0 +1,301 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Rename 'provider' to 'Publisher'
```

Comment:
Note that I intentionally let this drop all existing tables & recreate them instead of doing a data migration, because we have no data for these tables in production.

Reply:
My only thought is might be worth adding a "no downgrade" like other migrations ala

```
raise RuntimeError("Order No. 227 - Ни шагу назад!")
```

Reply:
No sense having a migration floating around that could potentially destroy data by downgrading the db?

Reply:
Good point! Added.